package.json

```diff
@@ -132,6 +132,20 @@
                     "default": "8080",
                     "description": "Specifies the port for the openHAB preview."
                 }
+                "openhab.username": {
+                    "type": [
+                        "string"
+                    ],
+                    "default": "",
+                    "description": "Specifies the username (optional) for accessing the openHAB preview/REST API"
+                }
+                "openhab.password": {
+                    "type": [
+                        "string"
+                    ],
+                    "default": "",
+                    "description": "Specifies the password (optional) for accessing the openHAB preview/REST API"
```

Comment:
`(optional) Specifies the Basic Auth password for accessing the openHAB preview/REST API`